Makefile

```diff
@@ -12,6 +12,10 @@ OUT_DIR=$(shell pwd)/build/.out
 
 .DEFAULT_GOAL := help
 
+AGENT_VERSION ?= 2.22.1
+ALPINE_VERSION ?= 3.16
+AGENT_PREFIX ?= agent
```

Comment:
I wonder if we should use `nginx` or `dataplane` in the sources (Makefile, kubernetes manifests, etc.). I think the main thing in that container is still NGINX, not the agent.

Reply:
Yeah, I struggled with this name. Dataplane seemed too generic, but naming it `nginx` seemed misleading since it's not just nginx running. I guess as long as the prefix is `nginx-kubernetes-gateway` the name `nginx` could work. Do you think it's possible that by naming it `nginx` users would think they can substitute the vanilla `nginx` image?

Reply:
maybe `nginx-with-agent`? a little verbose, but I think it describes unambiguously